src/app/contexts/ToggleContext/__snapshots__/index.test.jsx.snap

```diff
@@ -2,6 +2,6 @@
 
 exports[`ToggleContext should provide a toggles object 1`] = `
 <span>
-  {"toggleState":{"test":{"mpulse":{"enabled":false},"mediaPlayer":{"enabled":true},"chartbeatAnalytics":{"enabled":true}},"live":{"mpulse":{"enabled":false},"mediaPlayer":{"enabled":false},"chartbeatAnalytics":{"enabled":false}}}}
+  {"toggleState":{"test":{"mpulse":{"enabled":false},"mediaPlayer":{"enabled":true},"chartbeatAnalytics":{"enabled":true}},"live":{"mpulse":{"enabled":false},"mediaPlayer":{"enabled":false},"chartbeatAnalytics":{"enabled":true}}}}
```

Comment:
Nothing wrong with this PR, but maybe me implementing a snapshot test for this was not the best idea. means this snapshot will change and need to be updated everytime we change a toggle.

Reply:
Yeah, looks like the toggles code needs some more explicit tests. For the present would it be worth at least mocking out `lib/config/toggles` in the `ToggleContext` unit test with some dummy keys/values? At least then the snapshot won't keep needing to be updated when toggles change.

Reply:
That should work @jamesdonoh . I'll raise another issue to do that later.